<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="geometricMeanLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="averageLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="harmonicMeanLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="rmsLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Среднее арифметическое</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Среднее геометрическое</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Среднее гармоническое</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Среднее квадратическое</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addRowButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteRowButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>del</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="numbTable">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>190</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>23</number>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Введите числа:</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="tabValue">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>buttonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addRowButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>addRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteRowButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>delRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClick()</slot>
  <slot>addRow()</slot>
  <slot>delRow()</slot>
 </slots>
</ui>
